tests/test_accessory_driver.py

```diff
@@ -177,16 +177,7 @@ def setter_with_write_response(value=0):
         },
         "mock_addr",
     )
-    assert response == {
-        HAP_REPR_CHARS: [
-            {
-                HAP_REPR_AID: acc.aid,
-                HAP_REPR_IID: char_nfc_access_control_point_iid,
-                HAP_REPR_STATUS: 0,
-                HAP_REPR_VALUE: 1
-            },
-        ]
-    }
+    assert response is None
```

Comment:
Was this a bug?

Reply:
No, this was a change of behaviour made by me, I described it in a first post.

Original PR that added write responses could cause side effects for third-party code whose setters returned values on accident (when it wasn't a feature). In this PR I added a check that response value is only sent back if it was requested.